### PR TITLE
Include files unit tests in utils_test

### PR DESCRIPTION
**Pull request by frankfeng579**
- Fixed issues with files_test_helper
- Added inline functoin IsDirecotry().

### CODE REVIEW

File: base/cvd/cuttlefish/common/libs/utils/BUILD.bazel [hunk 1]
```diff
@@ -138,3 +140,23 @@ cc_test(
...
 )
+
+cc_library(
+    name = "files_test_helper",
+    testonly = 1,
+    srcs = [
+        "files_test_helper.cpp",
+    ],
+    hdrs = [
+        "files_test_helper.h",
```
Comment (Databean) [inline base/cvd/cuttlefish/common/libs/utils/BUILD.bazel:151]: Can `files_test_helper.cpp` and `files_test_helper.h` be part of the `utils_test` `srcs` member?

File: base/cvd/cuttlefish/common/libs/utils/files.h [hunk 2]
```diff
@@ -41,6 +41,9 @@ Result<void> CreateSymLink(const std::string& target, const std::string& link,
 bool FileHasContent(const std::string& path);
 Result<std::vector<std::string>> DirectoryContents(const std::string& path);
 bool DirectoryExists(const std::string& path, bool follow_symlinks = true);
+inline bool IsDirectory(const std::string& path) {
+  return DirectoryExists(path);
+};
```
Comment (Databean) [inline base/cvd/cuttlefish/common/libs/utils/files.h:46]: Is this new function necessary in this PR? I don't see it being used anywhere.

**Review by Databean — APPROVED** [review]

File: base/cvd/cuttlefish/common/libs/utils/BUILD.bazel (hunk 1 above)
Comment (frankfeng579) [reply base/cvd/cuttlefish/common/libs/utils/BUILD.bazel:151]: Tried that first but couldn't make it work. Bazel complains that files_test_helper.h can't be found (as included in files_test.cpp).

File: base/cvd/cuttlefish/common/libs/utils/files.h (hunk 2 above)
Comment (frankfeng579) [reply base/cvd/cuttlefish/common/libs/utils/files.h:46]: Discussed that with Jorge and he seems to be fine with this. The alternative is to add a comment to claim these two are the same.

File: base/cvd/cuttlefish/common/libs/utils/BUILD.bazel (hunk 1 above)
Comment (jemoreira) [reply base/cvd/cuttlefish/common/libs/utils/BUILD.bazel:151]: Probably because the `cc_test` is missing the `strip_include_prefix` attribute.

**Review by jemoreira — APPROVED** [review]

File: base/cvd/cuttlefish/common/libs/utils/BUILD.bazel [hunk 3]
```diff
@@ -129,6 +132,7 @@ cc_test(
         "-Icuttlefish",
     ],
     deps = [
+        # ":files_test_helper",
```
Comment (jemoreira) [inline base/cvd/cuttlefish/common/libs/utils/BUILD.bazel:135]: delete this

**Comment by frankfeng579** [comment]
Updated with suggestions from Databean.

**Pull request merged**